data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -153,6 +153,32 @@
     "Percent of individuals < 100% Federal Poverty Line in 2010"
 )
 
+# RSEI Aggregated Micro-data
+EPA_RSEI_NUMBER_FACILITIES_OUTPUT_FIELD = (
+    "Number of facilities affecting the tract"
+)
+EPA_RSEI_NUMBER_RELEASES_OUTPUT_FIELD = "Number of releases affecting the tract"
+EPA_RSEI_NUMBER_CHEMICALS_OUTPUT_FIELD = (
+    "Number of chemicals affecting the tract"
+)
+EPA_RSEI_AVERAGE_TOXICITY_OUTPUT_FIELD = (
+    "Average toxicity-weighted concentration of the cells in the tract"
+)
+EPA_RSEI_SCORE_OUTPUT_FIELD = (
+    "Sum of the risk-related scores of the cells in the tract"
+)
+EPA_RSEI_CSCORE_OUTPUT_FIELD = "Sum of the risk-related scores of the cells in the tract (Cancer toxicity weights)"
+EPA_RSEI_NCSCORE_OUTPUT_FIELD = "Sum of the risk-related scores of the cells in the tract (Noncancer toxicity weights)"
+EPA_RSEI_POPULATION_OUTPUT_FIELD = (
+    "Sum of the population of the cells in the tract"
+)
+
+EPA_RSEI_SCORE_PERCENTILE_RANK_FIELD = "Percentile rank for overall risk score"
+
+EPA_RSEI_SCORE_THRESHOLD_FIELD = (
+    "At or above 75 for overall percentile rank according to overall risk score"
```

Comment:
When this gets combined with all the other fields in the big `usa.csv`, "overall risk score" won't be descriptive enough for anyone to understand. Could you make this more descriptive? And please don't refer to percentile ranks, refer to percentiles.

Reply:
<img width="741" alt="Screen Shot 2022-01-13 at 1 17 58 PM" src="https://user-images.githubusercontent.com/94847739/149386680-bcdd3ce1-db59-47ef-a12b-b0e918890310.png">

The RSEI score is a function of all of these [inputs](https://www.epa.gov/rsei/understanding-rsei-results). I can come up with something to convey a more meaningful interpretation.